### DISCUSSION TITLE
How to obtain identity authentication for requests from other projects？

### DISCUSSION THREAD

**Discussion by tongtaott** [dexidp/dex#2908] [Q&A]
My project wants to access the interface of another project which use DEX for authentication, named Kubeflow. How can I obtain authentication? My development language is Java, and my development framework is springboot.Thank you for answering.

**Comment by sagikazarmark** [comment]
Dex exposes a standard OIDC interface, so anything that can interact with that should work.

**Reply by tongtaott** [reply]
Thank you for your answer. Do you know how to call the interface? Or, how can I confirm what the authentication interface address is?

**Reply by sagikazarmark** [reply]
It depends on how Dex is configured. Most probably, you need the issuer URL configured for Dex. The rest of the path is OIDC standard.
